### PR TITLE
Fix typo on README

### PR DESCRIPTION
**Pull request by elopio**
needs -> need

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ Once the pull request has been merged, `@miss-islington <https://github.com/miss
 will prepare the backport PR.
 
 If `@miss-islington <https://github.com/miss-islington>`_ encountered any issue while backporting,
-it will leave a comment about it. The PR will then needs to be backported manually.
+it will leave a comment about it. The PR will then need to be backported manually.
```
Comment (terryjreedy) [inline README.rst]: Or, 
`The PR then needs to be ...`
or 
`The PR should then be ...
I suspect the author switched constructions in mid sentence.

Comment (Mariatta) [reply README.rst]: Thanks! Yes, I prefer `The PR then needs to be .. `.

**Comment by elopio** [comment]
@Mariatta @terryjreedy thanks for the review. I've updated it to use your suggestion.

**Comment by Mariatta** [comment]
Thanks!

**Pull request merged**